web/templates/invite_user_modal.hbs

```diff
@@ -3,20 +3,25 @@
     <div class="alert" id="dev_env_msg"></div>
     {{/if}}
     <div class="input-group">
-        <label for="invitee_emails">{{t "Emails (one on each line or comma-separated)" }}</label>
-        <textarea rows="2" id="invitee_emails" name="invitee_emails" class="invitee_emails" placeholder="{{t 'One or more email addresses...' }}"></textarea>
-        {{#if is_admin}}
-        <div id="invite-method-choice">
-            {{t "or" }} <a role="button" tabindex="0" id="generate_multiuse_invite_button">{{t "Generate invite link" }}</a>
+        <label>{{t "How would you like to invite users?" }}</label>
+        <div class="invite_type_radio_section prop-element" id="invite-user">
+            <div id="generate_multiuse_invite_radio_container">
+                <label class="generate_multiuse_invite_radio_label display-settings-radio-choice-label">
```

Comment:
We can use some other class instead of `display-settings-radio-choice-label` here if we actually need a class here because this does not belong to the display settings section.